app/styles/molecules/button-dropdown.less

```diff
@@ -0,0 +1,62 @@
+.oss-button-dropdown {
+  position: relative;
+
+  &__trigger {
+    border: 1px solid @upf-primary-rock-blue;
+    border-radius: var(--border-radius-sm);
+    height: 37px;
+    padding: var(--spacing-px-6) 0;
+
+    div {
+      height: 37px;
+      padding: 0 var(--spacing-px-18);
+
+      &:last-child {
+        padding: 0 var(--spacing-px-12);
+        border-left: 1px solid @upf-primary-rock-blue;
+      }
+    }
+
+    i,
+    span {
+      color: var(--color-gray-900);
+    }
+  }
+
+  .oss-button-dropdown__items {
+    display: none;
```

Comment:
Handling this with `display: none` & `display: flex` in CSS makes the options always visible in the dom-elements. Is that wanted ?
Why not go with the `if` in the handlebars from ember ?